### PR TITLE
MMT-2197 - New Rake Task

### PR DESCRIPTION
**Pull request by jceaser**
adding a rake task to download the lattest CMR jar from the maven repository

### CODE REVIEW

File: lib/tasks/local_cmr.rake [hunk 1]
```diff
@@ -3,6 +3,41 @@ require 'test_cmr/load_data.rb'
...
+    # restore if error, as defined by there not being a jar
+    cmd_restore = 'cd cmr ; '\
+      "if [ -a \"#{jar_name}\" ] ; then "\
+        "rm #{jar_name}.oldest ; "\
+      'else '\
+        "mv -f #{jar_name}.last #{jar_name} ;"\
+        "mv -f #{jar_name}.oldest #{jar_name}.last ;"\
+        "rm #{jar_name}.tmp ;"\
+      'fi'
+    puts %x( #{cmd_restore} )
```
Comment (charleshuang80) [inline lib/tasks/local_cmr.rake]: I don't know what these outputs look like, are they sufficiently informative?

File: lib/tasks/local_cmr.rake [hunk 2]
```diff
@@ -3,6 +3,41 @@ require 'test_cmr/load_data.rb'
 namespace :cmr do
   include Cmr
 
+  jar_name = 'cmr-dev-system-0.1.0-SNAPSHOT-standalone.jar'
+  url_source = "https://maven.earthdata.nasa.gov/repository/mmt/#{jar_name}"
+
+  desc 'Fetch lattest CMR jar from the maven repository'
```
Comment (charleshuang80) [inline lib/tasks/local_cmr.rake]: wondering if we want to add anything to the readme for this task. not quite certain either way.

Comment (ryanmiller-1) [reply lib/tasks/local_cmr.rake]: I think it should be documented somewhere that we would point a new developer to.  Also, latest is misspelled in the description of the task

Comment (jceaser) [reply lib/tasks/local_cmr.rake:9]: I can add a section in the readme

File: lib/tasks/local_cmr.rake (hunk 1 above)
Comment (jceaser) [reply lib/tasks/local_cmr.rake:37]: I'll put a comment in on why I'm using puts %() and not Process.spawn()

File: lib/tasks/local_cmr.rake [hunk 3]
```diff
@@ -3,6 +3,45 @@ require 'test_cmr/load_data.rb'
 namespace :cmr do
   include Cmr
 
+  jar_name = 'cmr-dev-system-0.1.0-SNAPSHOT-standalone.jar'
+  url_source = "https://maven.earthdata.nasa.gov/repository/mmt/#{jar_name}"
+
+  desc 'Fetch latest CMR jar from the maven repository'
+  task :fetch do
+    # use puts %x() to run commands and not Process.spawn(), which returns
+    # imediatly, because curl can take a while and we want to see the output
```
Comment (charleshuang80) [inline lib/tasks/local_cmr.rake:12]: typo on immediately (not important though).

**Pull request merged**